lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -61,6 +61,24 @@ mapped_vlan:
     config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
     default_value: ''
 
+mt_full_support:
+  # This is only used for determining support for Multi-Tenancy Full
+  kind: boolean
+  /N7/:
+    default_only: true
+  else:
+    # this feature is always off on these platforms and cannot be changed
+    default_only: false
+
+mt_lite_support:
```

Comment:
Here's the license guide:
http://www.cisco.com/c/en/us/td/docs/switches/datacenter/sw/nx-os/licensing/guide/b_Cisco_NX-OS_Licensing_Guide/b_Cisco_NX-OS_Licensing_Guide_chapter_01.html

For now though, I suggest to add a story to backlog for adding mt-lite puppet support for 5/6k. We can look at it when we test 5/6k